p2p/discovery/database.go

```diff
@@ -22,10 +22,11 @@ func NewDatabase() *database {
 	}
 }
 
-func (db *database) add(id common.Hash, value *Node) {
+func (db *database) add(value *Node) {
 	db.mutex.Lock()
 	defer db.mutex.Unlock()
-	db.m[id] = value
+	sha := value.getSha()
```

Comment:
nice to add new line between #27 and # 28

Reply:
sure